### ISSUE TITLE
macOS 10.14 (Mojave) - BetterTTV no longer installable on Safari

### ISSUE THREAD

**Issue by Wilkolicious** [night/BetterTTV#3158]
Safari Extensions can no longer be installed from outside the App Store / Safari Extensions Gallery on Safari 12.0.

The only work-around seems to be through the extension builder.  Details here: https://georgegarside.com/blog/macos/install-any-safari-extension-macos-mojave/

**Comment by night** [comment]
ye, lets pay $99/yr to make the extension available to safari users (which is like nobody, no offense)

**Comment by LeotomasMC** [comment]
Safari is the Internet Explorer of MacOS, Heck apple even used IE on the earlier Macs before they had Safari, so its not surprising that Safari is a bad web browser, although its better than IE.

Edge is probably the best default Web Browser ive seen on an OS (other than Firefox on most Linux Distros)

**Comment by Juice805** [comment]
FWIW I use it in Safari. It's great. Definitely better than Edge.

**Comment by AdamChandler1986** [comment]
I know Safari is not important. I hope you'll re-consider supporting this in the future. Safari just removed your extension from my browser. technical Mac users use Safari because it has the lowest resource use of any other browser. Chrome & FireFox cut our battery life in half and Chrome is very user-privacy-hostile to a point that privacy-conscious users choose Safari. I wish Apple didn't gate-keep access to its users with a $99 tax which you and I know is completely wrong for them to do.

**Comment by dljsjr** [comment]
I'd also like to throw my hat in to the ring for continued Safari support. I know that it's a smaller browser market and the $99 toll bridge is definitely a developer hostile move. But my daily driver is a MacBook and I use Safari for its integration with the rest of my iOS devices and the battery life/memory use concerns.

I'd be more than happy to contribute to a Patreon or something if that was what was needed to keep Safari support alive.

That said as a developer myself I'm sympathetic to the idea that you wouldn't want to support Safari as a matter of principle. I just wanted to chime in and express interest in keeping support alive.

**Comment by Vardek13** [comment]
Hi,

Related to this issue, FrankerfaceZ found a workaround through Tampermonkey, so if you got that extension, safari users would only need the user script in order to use the extension like it was before. I am not very knowledgable on the subject, but is the user script for betterttv accessible already? If not is it possible to make it accessible?

Thanks in advance.

**Comment by MysticalOS** [comment]
Yeah I actually used that work around for some other stuff too. Like socialfixer has a monkey script you can use instead of extension. It works as a work around since Tampermonkey is in the gallery so can be used with safari 12.0

**Comment by samfun123** [comment]
Here is a [userscript](https://gist.github.com/samfun123/8541e9d78f0f9aee5fd4c262b7ebc95e/raw/04ab5d00aa16ffeb843683f78b88af7de9e80333/BetterTTV.user.js) which should do the same thing as the chrome extension. [Source](https://gist.github.com/samfun123/8541e9d78f0f9aee5fd4c262b7ebc95e)

**Comment by Vardek13** [comment]
Thank you that worked perfectly! Honestly, I think this is the better solution for everybody.

**Comment by isp13** [comment]
https://github.com/isp13/Safari_BetterTTV
